editor/header/preview-button/index.js

```diff
@@ -73,6 +73,16 @@ export class PreviewButton extends Component {
 			'about:blank',
 			this.getWindowTarget()
 		);
+
+		const popupLoader = renderToString(
+			<div>
+				<p>Please wait&hellip;</p>
+				<p>Generating preview.</p>
+			</div>
+		);
+		const css = '<style type="text/css"> div { margin-top: 25%; } p { text-align: center; } </style>';
```

Comment:
Is the intention that this be centered? Seems like flexbox might serve better, if so.